src/services/NotificationService/NotificationService.js

```diff
@@ -1,96 +1,112 @@
+import axios from 'axios';
+import { userSessionData } from '../../settings/constants';
+import { notificationUrl } from '../../settings/constants';
 
-import axios from "axios";
-import {userSessionData} from '../../settings/constants';
-import {notificationUrl} from '../../settings/constants';
+const _URL = notificationUrl;
 
-const _URL=notificationUrl;
+const _getNotification =
+  _URL + '/getNotification/' + sessionStorage.getItem('id');
 
- const _getNotification=_URL+'/getNotification/'+sessionStorage.getItem("id");
+const _markedReadNotification = _URL + '/markedReadNotification/';
+const _markedReadRegularizationNotification =
+  _URL + '/markAllAsReadNotification/';
 
- 
+const _getAllNotification =
+  _URL + '/getAllNotification/' + sessionStorage.getItem('id');
 
+const _markAllReadAsNotification = _URL + '/markAllAsReadNotification/';
 
-const _markedReadNotification=_URL+'/markedReadNotification/';
-const _getAllNotification=_URL+'/getAllNotification/'+sessionStorage.getItem("id");
+export function getData() {
+  const token = localStorage.getItem('jwt_token');
 
+  const config = {
+    headers: {
+      Authorization: `Bearer ${token}`,
+      Accept: 'application/json',
+      'Content-Type': 'application/json'
+    }
+  };
+  return axios.get(_URL, config);
+}
+
+export function postData(payload) {
+  const token = localStorage.getItem('jwt_token');
```

Comment:
store token in function scope not in block scope

Reply:
resolved